appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts

```diff
@@ -89,32 +66,9 @@ describe('SETUP: ', () => {
         }
       };
       await MailFolderScreen.checkInboxScreen();
-      await MailFolderScreen.clickCreateEmail();
-      await NewMessageScreen.setAddRecipient(userEmail);
-      await NewMessageScreen.checkAddedRecipient(userEmail);
-      await NewMessageScreen.clickBackButton();
-
-      // Go to Contacts screen
-      await MenuBarScreen.clickMenuIcon();
-      await MenuBarScreen.checkUserEmail();
-
-      await MenuBarScreen.clickSettingsButton();
-      await SettingsScreen.checkSettingsScreen();
-      await SettingsScreen.clickOnSettingItem('Contacts');
-
-      await ContactScreen.checkContactScreen();
-      await ContactScreen.checkContact(userEmail);
-      // Go to Contact screen
-      await ContactScreen.clickOnContact(userEmail);
-      await ContactPublicKeyScreen.checkPgpUserId(userEmail);
-      await ContactPublicKeyScreen.checkPublicKeyDetailsNotEmpty();
-      await ContactPublicKeyScreen.clickOnFingerPrint();
+      await PublicKeyHelper.checkSignatureAndFingerprints(userEmail, newSignatureDate, newFingerprintsValue);
```

Comment:
`await MailFolderScreen.checkInboxScreen()` should be included in `checkSignatureAndFingerprints` too, as it's performed each time

Reply:
ok, pushed